java/README.md

```diff
@@ -1,3 +1,8 @@
 # Java Code
 
-Top-level directory for all Java code.
+Top-level directory for all Java code. To build and test:
+
+```bash
+nix develop
```

Comment:
(nit) This is always assumed, no other script would work without first being in the nix shell anyways, so I'd leave it out (otherwise we should add it everywhere else for consistency).

Reply:
Done.